src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -4532,7 +4534,7 @@ lElseClause:
                         allConvertibleToObject = False
                     End If
 
-                    ' this will pick up lambdas which couldn't be inferred, and array literals which lacked a dominant type,
+                    ' this will pick up lambdas which couldn't be inferred, typeless tuples, and array literals which lacked a dominant type,
```

Comment:
>typeless tuples [](start = 76, length = 15)

The change to the comment doesn't look accurate, all tuple literals are handled above. It also looks like the original comment was incorrect about lambdas and array literals as well.